tests/Fixer/Phpdoc/PhpdocTrimAfterDescriptionFixerTest.php

```diff
@@ -0,0 +1,193 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\Phpdoc;
+
+use PhpCsFixer\Tests\Test\AbstractFixerTestCase;
+
+/**
+ * @author Nobu Funaki <nobu.funaki@gmail.com>
+ *
+ * @internal
+ *
+ * @covers \PhpCsFixer\Fixer\Phpdoc\PhpdocTrimAfterDescriptionFixer
+ */
+final class PhpdocTrimAfterDescriptionFixerTest extends AbstractFixerTestCase
+{
+    public function testFix()
```

Comment:
could you please rearrange tests? 

do one method that do testing (so `testFix`), but with data provider instead of multiple testing methods.

your data provider would look like
```
return [
    'case foo' => [$expected1, $input1],
    'case bar' => [$expected2, $input2],
    ...,
];
```